commit e14c8ad5c45a43af423e19667e60f4fb54836fe1 (refs/changes/46/1046/1)
Author: Orlando Vazquez <ovazquez@gmail.com>
Date:   2016-12-02T00:28:28-08:00 (2 years, 10 months ago)
    
    HEAD-2333 joysetup.sh script should log outside of /tmp as subsequent reboots will cause this file to be lost

diff --git a/scripts/joysetup.sh b/scripts/joysetup.sh
index e8dc8eda..a7ff85f4 100755
--- a/scripts/joysetup.sh
+++ b/scripts/joysetup.sh
@@ -33,8 +33,8 @@ MIN_DISK_TO_RAM=2
 # status output goes to /dev/console instead of stderr
 exec 4>/dev/console
 
-# keep a copy of the output in /tmp/joysetup.$$ for later viewing
-exec > >(tee /tmp/joysetup.$$)
+# keep a copy of the output in /var/tmp/joysetup.$$ for later viewing
+exec > >(tee /var/tmp/joysetup.$$)
 exec 2>&1
 
 set -o errexit
